package.json

```diff
@@ -6,9 +6,9 @@
   "scripts": {
     "start": "nodemon lib/index.js --exec babel-node",
     "build": "babel lib -d dist",
-    "lint": "eslint lib/",
+    "lint": "./node_modules/.bin/eslint lib/",
```

Comment:
Hard-coded paths to the executables isn't needed in the context of an NPM script. It's done automatically if it's at the start of a script